gdal/frmts/tiledb/tiledbdataset.cpp

```diff
@@ -1904,20 +1919,24 @@ TileDBDataset::Create( const char * pszFilename, int nXSize, int nYSize, int nBa
         GDALDataType eType, char ** papszOptions )
 
 {
+    const char* pszBlockXSize = CSLFetchNameValue( papszOptions, "BLOCKXSIZE" );
```

Comment:
frmts/tiledb/tiledbdataset.cpp:1922,style,unreadVariable,Variable 'pszBlockXSize' is assigned a value that is never used.

Reply:
Thanks, leftover debug. I have updated the PR.